### ISSUE TITLE
Input data prepare

### ISSUE THREAD

**Issue by ytang0831** [GET-Foundation/get_model#34]
Hi Xi，

I tried to run your model with my own data. During this process, I noticed that the normlaization methods of TPM in prepare_pbmc.ipynb and prepare_pbmc_recall_peak.ipynb were different. TPM/TPM.max() was performed in prepare_pbmc.ipynb, but this processing was not done in prepare_pbmc_recall_peak.ipynb. May I ask if performing this processing has any impact during training and prediction

Thanks！

**Comment by fuxialexander** [comment]
I was experimenting with that in prepare_pbmc_recall_peak.ipynb. The default behavior and what the pretrained model expect is the `TPM/TPM.max()` normalized version. But if you finetune the model with the unnormalized version I think it would still work.
